2024-12-10 02:18:20,868 - CRITICAL - Connection to https://api.openweathermap.org/data/2.5/weather?lat=49.15&lon=9.21&appid=8835ebfe28f356c4b8b2b928a9898fc6&units=metric could not be established. Error: HTTPSConnectionPool(host='api.openweathermap.org', port=443): Max retries exceeded with url: /data/2.5/weather?lat=49.15&lon=9.21&appid=8835ebfe28f356c4b8b2b928a9898fc6&units=metric (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0xf67709d0>, 'Connection to api.openweathermap.org timed out. (connect timeout=None)'))
2024-12-10 02:18:20,869 - ERROR - Returned weather data was null.
2024-12-10 02:18:20,869 - ERROR - API returned no value. Error: <class 'customExceptions.DataMissingPublicAPIError'>
